tools/wptserve/wptserve/sslutils/__init__.py

```diff
@@ -1,9 +1,7 @@
 # flake8: noqa (not ideal, but nicer than adding noqa: F401 to every line!)
```

Comment:
@gsnedders you added this; do you remember why you needed to keep the openssl/pregenerated imports?

Either we should remove the noqa annotation, or we should keep the imports. (I'd merge immediately if you did the latter; need more info to approve the former.)

Reply:
I have no idea. I was trying to change as little as possible.

Reply:
>was trying to change as little as possible.

It seems okay to merge then 😁

Reply:
I offered two options in my comment; please implement one of them.

Reply:
Done!